plantcv/plantcv/hyperspectral/read_data.py

```diff
@@ -184,9 +183,6 @@ def read_data(filename):
     pseudo_rgb = _make_pseudo_rgb(spectral_array)
     spectral_array.pseudo_rgb = pseudo_rgb
 
-    if params.debug == "plot":
-        plot_image(pseudo_rgb)
-    elif params.debug == "print":
-        print_image(pseudo_rgb, os.path.join(params.debug_outdir, str(params.device)+"_pseudo_rgb.png"))
+    _debug(visual=pseudo_rgb, filename=os.path.join(params.debug_outdir, str(params.device)+"_pseudo_rgb.png"))
```

Comment:
```suggestion
    _debug(visual=pseudo_rgb, filename=os.path.join(params.debug_outdir, str(params.device) + "_pseudo_rgb.png"))
```

Add spaces between the strings and + operator